### DISCUSSION TITLE
How to change cursorline color?

### DISCUSSION THREAD

**Discussion by zhu-jiyuan** [ibhagwan/fzf-lua#1326] [Q&A] [answered]
Hello everyone, fzf-lua is very easy to use. Recently I used bat-native to replace the preview of neovim. But when I use fzf-lua rg, I can't see the cursorline clearly. This is very painful for me.
![image](https://github.com/user-attachments/assets/d9c0db48-5196-48f7-a1ab-30bf665602db)

**Comment by ibhagwan** [comment]
The cursor line background and foreground are defined by `fg+` and `bg+` colors of fzf which you can set via `fzf_colors`, alternatively you can set `fzf_colors=true` to have fzf-lua generate the fzf color scheme for you and then modify `FzfLuaFzfCursorLine` if you don’t like the default.

https://github.com/ibhagwan/fzf-lua?tab=readme-ov-file#fzf-colors

**Reply by ibhagwan** [reply, marked as answer]
As for the bat cursor line color, it is defined by your selected bat color scheme, you can modify bat’s theme by setting `previewers.bat.theme` (see the README default options section), run `bat --list-themes` to see all available themes.

**Reply by zhu-jiyuan** [reply]
thank you! I tried changing the colors and finally found out it was a problem with the PowerShell theme. 🤣
